### ISSUE TITLE
tasks not working as before

### ISSUE THREAD

**Issue by tionkje** [Microsoft/vscode#185]
In vscode 9 i had setup a simple task to rsync my project to a server.

tasks.json
```json
{
    "version": "0.1.0",
    "command": "zsh",
    "isShellCommand": true,
    "args":["-c"],
    "tasks": [
      {
        "taskName": "up",
        "args": ["rsync -avicCFF --no-t --no-p --no-g ./ bastiaan@front:~/front/; echo DONE;"],
        "isBuildCommand": true
      }
    ]
}
```

It worked perfectly, now after the update i only get 
```
zsh:1: command not found: up
```
It seems its adding the taskname as argument instead of the args array.
What changed in tasks.json that this does not work anymore?

**Comment by tionkje** [comment]
I looked at
https://code.visualstudio.com/docs/editor/tasks_appendix

and apparently some default changed or was added
i needed
"suppressTaskName":true,


so correct tasks.json
```json
{
    "version": "0.1.0",
    "command": "zsh",
    "isShellCommand": true,
    "args":["-c"],
    "suppressTaskName":true,
    "tasks": [
      {
        "taskName": "up",
        "args": ["rsync -avicCFF --no-t --no-p --no-g ./ bastiaan@front:~/front/; echo DONE;"],
        "isBuildCommand": true
      }
    ]
}
```

**Comment by dbaeumer** [comment]
@tionkje actually there was a bug how we constructed the command line in 0.9.0. What we did in 0.9.0 was the follow ${command} ${command args} ${tasks args} ${taskname}. But the task args should come after the task name. So we fixed that for 0.10.0 and the order is now: ${command} ${command args} ${taskname} ${tasks args}. As you found out you can use "suppressTaskName" to not add the task name to the command line.

**Comment by JohnW71** [comment]
Glad to finally find an explanation for this, been going mad wondering why my tasks all broke. Now up and running again.
